docs/DEV_ENV.md

```diff
@@ -1,72 +1,82 @@
 # Local Development Environment
 
-This uses the [Happy Path setup](https://wiki.czi.team/display/TECH/Data+Portal+Happy+Path+--+How+it+works) created by the shared infra team. 
```

Comment:
prettier formatting in this file! Only real change is adding `pre-commit` install step below